packages/backend-lib/src/server/genericErrorMiddleware.ts

```diff
@@ -19,11 +18,16 @@ export interface GenericErrorMiddlewareCfg {
   formatError?: (err: ErrorObject) => void
 }
 
+export interface ErrorReportingService {
+  // biome-ignore lint/suspicious/noExplicitAny: it is expected of the `captureException` to be tolerant of any argument
```

Comment:
fyi: this rule is already excluded in our biome settings. Maybe they're just not picked up by your IDE.
Au-contraire: I expect linters to complain when we unnecessarily exclude rules (eslint does that, maybe not Biome).

Maybe the linters setup in the monorepo is not great yet (run `yarn check` in the monorepo root if unsure - it'll check **everything**)

Reply:
After I wrote that message - I realized that indeed `pn check` was not good enough in this monorepo. It was doing `build`, but not `typecheck`.

If you pull from main - now `pnpm check` is doing everything it should 👍